### ISSUE TITLE
'QUEUED' status?

### ISSUE THREAD

**Issue by stuartlewis** [DataVault/datavault#114]
At present, a deposit in the queue waiting for a worker has the state of 'NOT STARTED'.  Could we add an additional status of 'QUEUED' so we know it has got as far as the message queue?

**Comment by tomhigginsuom** [comment]
I'm not sure how different this would be to the "NOT STARTED" status because the broker request handler puts it on the queue immediately (so "NOT STARTED" is effectively the same as "QUEUED"). Perhaps we should rename "NOT STARTED" to "QUEUED"? I also think we shouldn't be displaying raw statuses to users - perhaps we can have a friendlier indicator.

**Comment by stuartlewis** [comment]
Yes, that might be one option.  I think QUEUED might be better for the user, as they can see something has happened, and it is in a waiting state.

The only benefit of adding QUEUED as a new state, is it might highlight to us if the MQ has died, as a task wouldn't have been able to move to the QUEUED state, ans will still be NOT_STARTED.
